src/poetry/console/commands/env/activate.py

```diff
@@ -47,7 +47,11 @@ def _get_activate_command(self, env: Env) -> str:
         elif shell in ["csh", "tcsh"]:
             command, filename = "source", "activate.csh"
         elif shell in ["powershell", "pwsh"]:
-            command, filename = ".", "Activate.ps1"
+            if WINDOWS:
+                command, filename = ".", "Activate.ps1"
+            else:
+                command, filename = ".", "activate.ps1"
+
```

Comment:
I think we can just make it lower case. Actually, it seems it is also lower case on Windows (but it does not matter because Windows does not care about case).